src/components/debug_config/LogCard.vue

```diff
@@ -47,16 +85,26 @@ export default {
     return {
       logData: "-- noch nicht geladen --",
       loading: false,
+      foundFiles: [], // Array to store found files with title, suffix, and description
+      selectedVariant: "", // Selected file variant
+      copyMessage: "", // Message to show when log data is copied
     };
   },
+  mounted() {
+    this.checkLatestLog(this.logFile);
+  },
```

Comment:
Das hat zur Folge, dass beim Laden der Seite pro LogCard mindestens zwei HTTP-Aufrufe gemacht werden, welche vermutlich nicht notwendig sind. Je nach Größe der Logdaten und Qualität der Verbindung kann der Abruf durchaus für Verzögerungen sorgen, daher sollte das nicht in `mounted()` gemacht werden.
```suggestion
```
Ich würde vorschlagen, dass die Komponente `openwb-base-card` um Events `expanded` und `collapsed` erweitert wird, sodass man erkennen kann, ob die Karte aufgeklappt wurde. Dann kann hier z.B. mit `@expanded` darauf reagiert werden.

Reply:
ist umgesetzt.